other/sortPeopleName.js

```diff
@@ -0,0 +1,39 @@
+// Cara menyortir nama orang menggunakan Javascript
+
+// Data nama didapatkan dari halaman demo Faker.js
+// https://rawgit.com/Marak/faker.js/master/examples/browser/index.html
+// Faker - https://github.com/marak/Faker.js
+const daftarNama = [
+  "Jarret Sudrajat",
+  "Demond Usada",
+  "Ariel Safitri",
+  "Jimmy Febian",
+  "Elfrieda Elvina",
+  "Wiley Satria",
+  "Abbie Sudiati",
+  "Susie Pradana",
+  "Clarabelle Kusmawati"
+];
+
+// Sortir dari nama yang depannya huruf a yang di awal
+const dariASampaiZ = [...daftarNama].sort((a, b) => a.localeCompare(b));
+
+// Sortir dari nama yang depannya huruf z yang di awal
+const dariZSampaiA = [...daftarNama].sort((a, b) => b.localeCompare(a));
+
+console.log(dariASampaiZ);
+console.log(dariZSampaiA);
+
+/**
+ * Penjelasan code:
+ * Prototype string localeCompare adalah fitur javascript yang
+ * fungsinya membandingkan string awal dengan string yang dijadikan
+ * parameter. Fungsi ini akan mereturn -1, 1 atau 0, hal ini bisa
+ * dikombinasikan dengan fungsi sort array
```

Comment:
Mungkin bisa ditambahkan penjelasan tentang prototype `sort`-nya juga
```suggestion
 * Prototype array sort merupakan sebuah method yang digunakan
 * untuk mengurutkan isi dari sebuah array. Method ini menerima
 * sebuah argumen berupa function yang memiliki dua parameter,
 * yaitu: nilai pertama (a) & nilai kedua (b).
 * Jika function tersebut mereturn nilai lebih dari 0, maka b diletakkan sebelum a.
 * Jika function tersebut mereturn nilai kurang dari 0, maka a diletakkan sebelum b.
 * Jika function tersebut mereturn nilai 0, maka a dan b tidak diubah urutannya.
 *
 * Prototype string localeCompare adalah fitur javascript yang
 * fungsinya membandingkan string awal dengan string yang dijadikan
 * parameter. Fungsi ini akan mereturn -1, 1 atau 0, hal ini bisa
 * dikombinasikan dengan fungsi sort array
```

Reply:
Setelah melihat kode yang lain, sepertinya penjelasan yang ini lebih cocok dipisah? 🤔

Reply:
Cocoknya sih dipisah, soalnya ini yang jadi bahasan kan si `localeCompare`, mungkin ini jadi masuknya other. Si fungsi `sort` di basicnya